CHANGES.md

```diff
@@ -40,6 +40,12 @@
     A wrapper around the 'ReadP' parser combinator, providing behaviour
     that's closer to the more popular parser combinator libraries.
 
+  * `XMonad.Hooks.WorkspaceScreen`
```

Comment:
This seems more general than just X.H.StatusBar to me.

Reply:
I was considering using the StatusBar namespace more. To be honest, I don't know why ClickableWorkspaces is under Util

Reply:
> To be honest, I don't know why ClickableWorkspaces is under Util

Because when reviewing that PR many months ago, I didn't think about this at all :-)

Reply:
It may be more general than X.H.StatusBar but I still think that this would be the overwhelming use-case; may be worth moving it there anyways

Reply:
Maybe we need clearer rules for each of the top-level modules

Reply:
I've been thinking about this, and maybe this should go under `X.H.StatusBar`? All functions return `X PP`, and `PP` lives in that namespace

Reply:
> I've been thinking about this, and maybe this should go under `X.H.StatusBar`? All functions return `X PP`, and `PP` lives in that namespace

Yeah, let's do this :-)

Reply:
:+1: Now that we have this "namespace" it'd be weird not to use it